client/src/main/java/pro/panopticon/client/sensor/impl/WarnModeSuccessrateSensor.kt

```diff
@@ -0,0 +1,15 @@
+package pro.panopticon.client.sensor.impl
+
+import pro.panopticon.client.util.NowSupplier
+
+/**
+ * This sensor is only capable of giving warnings, and will never result in an alert to pagerduty
+ */
+open class WarnModeSuccessrateSensor : SuccessrateSensor {
+    constructor(numberToKeep: Int, warnLimit: Double?) : super(numberToKeep, warnLimit, 1.1) {}
+    internal constructor(numberToKeep: Int, warnLimit: Double?, nowSupplier: NowSupplier?) : super(numberToKeep,
```

Comment:
Trenger vi to konstruktører da, eller er det mest for å sikre oss at ikke altfor mye brekker i de ulike mikrotjenestene når de tar i bruk nyeste versjon. `nowSupplier` kan feks default settes til null i konstruktøren.